packages/config/config/devices/0x027a/zen26_1.0-1.0.json

```diff
@@ -50,37 +61,59 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "LED on when switch off",
+					"label": "LED is on when switch is off",
 					"value": 0
 				},
 				{
-					"label": "LED on when switch on",
+					"label": "LED is on when switch is off",
 					"value": 1
 				},
 				{
-					"label": "LED off",
+					"label": "LED is always off",
 					"value": 2
 				},
 				{
-					"label": "LED on",
+					"label": "LED is always on",
 					"value": 3
 				}
 			]
 		},
 		"3": {
 			"label": "Auto Turn-Off Timer",
-			"description": "enable the Auto Turn-Off Timer",
-			"valueSize": 2,
+			"description": "Enable the auto turn-off timer.",
+			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 32767,
+			"maxValue": 1,
 			"defaultValue": 0,
 			"readOnly": false,
 			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"4": {
+			"label": "Auto Turn-Off Timer Duration",
+			"description": "Set the number of seconds to automatically turn off switch once it has been turned on.",
```

Comment:
This description is unnecessary and can be removed. If the label is clear and there is a unit defined you don't need it.

Reply:
Excellent! I thought description was a mandatory field for some reason, and a lot of them, my description basically was "does the thing in the label". Description being optional will definitely help make a lot of this cleaner.

Reply:
Yeah, that's a recent change. It helps a lot on the ui itself too.

Reply:
Actually, that was always optional:
https://zwave-js.github.io/node-zwave-js/#/development/config-files?id=paraminformation
We just had a description for almost every parameter like forever, so it might not seem that way.